application/controllers/collection_instrument.py

```diff
@@ -96,6 +95,9 @@ def upload_seft_to_bucket(self, exercise_id, file, ru_ref=None, classifiers=None
         survey = self._find_or_create_survey_from_exercise_id(exercise_id, session)
         survey_id = survey.survey_id
         survey_service_details = get_survey_details(survey_id)
+        if survey_service_details["surveyMode"] == "EQ_AND_SEFT" and ru_ref is not None:
+            raise RasError("Can't upload an reporting unit specific instrument for an EQ_AND_SEFT survey", 400)
```

Comment:
"a" instead or "an"

Reply:
Doh!  I went back and fourth on a few different wordings and missed that.  Thanks for catching it ^^